src/containers/articles/Article.js

```diff
@@ -14,29 +15,39 @@ class ArticlePage extends Component {
   }
 
   handleEditClick = () => {
-    const { articleData, getArticle, history } = this.props; // eslint-disable-line
+  const { articleData, getArticle, history } = this.props; // eslint-disable-line
     getArticle(articleData.slug);
     history.push('/articles/update');
```

Comment:
This action should occur when this `getArticle(articleData.slug);` is successful, your implementation has not catered that?

Reply:
The article page cannot be loaded unless article data has been found 
If the article data is not there then that page will remain to be in a loading state and thus neither the article nor the buttons would show up.